aurget

```diff
@@ -29,6 +30,9 @@ Usage: aurget [ -h | -S* [ --options ] [ -- ] <arguments> ]
         -Sp  <package>  print the PKGBUILD for <package>
         -Si  <package>  print extended info for <package>
 
+        --aur3          use the AUR3 ($AUR3)
```

Comment:
I was actually thinking similar thoughts.
I didn't like the two aur{3,4}_packages_url functions anyway, just did it because I thought this would be closer to you function-style.
$AUR3 also did not exist first and was only created to avoid confusion. However, as it stays the default, I think it makes sense to keep it as $AUR.

This way this will be more like a workaround for AUR4, which does somehow make sense, since it won't last for long.

So do as you want.